part_1/src/com/chernova/library/Library.java

```diff
@@ -0,0 +1,137 @@
+package com.chernova.library;
+
+import java.util.*;
+
+public class Library {
+    public static LinkedList<Book> list = new LinkedList<>();
+    public static final String ANSI_GREEN = "\u001B[32m";
+    public static final String ANSI_RED = "\u001B[31m";
+    public static final String ANSI_YELLOW = "\u001B[33m";
+    public static final String ANSI_RESET = "\u001B[0m";
+
+
+    //получить все книги (метод ВОЗВРАЩАЕТ список всех книг в библиотеке)
+    public static LinkedList<Book> getAllBooks() {
+        if (list.isEmpty()) {
+            System.out.println("Библиотека пуста");
+        }
+        return list;
+    }
+
+
+    //  добавить книгу (принимает объект книги и добавляет его в список товаров)
+    public static void addBook(Book book) {
+        boolean isExistBook = false;
+        for (int i = 0; i < list.size(); i++) {
+            if (list.get(i).getId() == book.getId()) {
+                isExistBook = true;
+                break;
+            }
+        }
+        if (isExistBook) {
```

Comment:
and what? A lot of libraries has more than 1 similar books. Because of that we have ID, Date and other fields that tells us about similarity of books. We need to have possibility to add as many as we want similar books, but they should differs by IDs and Dates